Utilities/build-script-helper.py

```diff
@@ -9,6 +9,32 @@
 import shutil
 import subprocess
 import sys
+import json
+
+def error(message):
+  print("--- %s: error: %s" % (os.path.basename(sys.argv[0]), message))
+  sys.stdout.flush()
+  raise SystemExit(1)
+
+def get_build_target(swift_exec, args, cross_compile=False):
+  """Returns the target-triple of the current machine or for cross-compilation."""
+  try:
+    command = [swift_exec, '-print-target-info']
+    if cross_compile:
+      cross_compile_json = json.load(open(args.cross_compile_config))
+      command += ['-target', cross_compile_json["target"]]
```

Comment:
These three lines and the `cross_compile` flag can be removed as they are unused on macOS. With SPM, [this flag was only set for Android to get the target directory](https://github.com/apple/swift-package-manager/blob/main/Utilities/bootstrap#L221), which is unneeded here.